app/upgrades/v11/README.md

```diff
@@ -0,0 +1,2 @@
+# Upgrade v10 Changelog
+1. IBCv7 version bump ([#765](https://github.com/org/repo/pull/765))
```

Comment:
```suggestion
# Upgrade v11 Changelog
1. IBCv7 version bump ([#840](https://github.com/Stride-Labs/stride/pull/840))
```

I'm just realizing the v10 change log has github.com/org/repo instead of Stride-Labs/stride lol. Mind updating that when you make the changelog PR after this?